prereise/gather/winddata/hrrr/downloader.py

```diff
@@ -0,0 +1,13 @@
+import shutil
+
+import requests
+
+
+class Downloader:
+    def download(self, url, file_path, headers={}, flags="ab"):
```

Comment:
It seems `download` could be a static method

Reply:
thanks!